src/utils/index.ts

```diff
@@ -39,6 +39,7 @@ export const validateOptions = (options?: Options): ValidatedOptions => {
     projectId: options.projectId,
     isTestnet: options.isTestnet,
     version: options.version || DEFAULT_API_VERSION,
+    http2: options.http2 || true,
```

Comment:
there is no way to disable http2 if it always fallbacks to true
`??` is your friend!